pybamm/spatial_methods/finite_volume.py

```diff
@@ -146,11 +146,29 @@ def divergence(self, symbol, discretised_symbol, boundary_conditions):
                 out = divergence_matrix @ ((r_edges**2) * discretised_symbol)
             elif submesh.coord_sys == "cylindrical polar":
                 out = divergence_matrix @ (r_edges * discretised_symbol)
-        else:
+        elif submesh.coord_sys == "cartesian":
             out = divergence_matrix @ discretised_symbol
-
+        elif submesh.coord_sys == "none":
+            # what should we do now
+            print("It is none!")
+            out = divergence_matrix @ discretised_symbol
```

Comment:
I have a question here actually. if the coord_sys is none, what do we do for `out = divergence_matrix @ discretised_symbol`? @agriyakhetarpal

Reply:
I haven't worked with the spatial methods before haha, perhaps @brosaplanella can answer this better

However, here, the coordinate system of the submesh should not be compared with `"none"` (which is a `str`) but rather with `None` (a null object in Python).

Reply:
I took his comment to mean that None was already an acceptable option since it would default to Cartesian. I have not worked on it to double check though.

So something sort of like this instead of adding to "KNOWN_COORD_SYS".
```python
if coord_sys is not None and coord_sys not in KNOWN_COORD_SYS:
            raise ValueError(
                f"Coordinate system is {coord_sys}, not in cartesian, cylindrical polar, spherical polar, None")
```

Reply:
I think `None` does not need to be caught here (as if there is no `coord_sys` we do not need to discretise). You just need to catch it as an exception at `independent variable.py`. I think that should make the tests pass.